spec/spec_helper.rb

```diff
@@ -9,10 +9,21 @@
 require 'factories'
 require 'database_cleaner'
 require 'rspec/autorun'
+require 'pry-nav'
+require 'pry-rails'
+require 'pry-stack_explorer'
+require 'pry-theme'
+require 'shoulda-matchers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.